README.md

```diff
@@ -80,6 +80,16 @@ Install this project with:
    npm install
 ```
 
+### Run tests
+
+To run tests, First create a directory, initialize npm, install webpack locally and then follow the steps:
+
+   - [Basic Setup](https://webpack.js.org/guides/getting-started/#basic-setup)
+   - [Setting up HtmlWebpackPlugin](https://webpack.js.org/guides/output-management/#setting-up-htmlwebpackplugin)
+   - [Loading CSS](https://webpack.js.org/guides/asset-management/#loading-css)
+   - [Using webpack-dev-server](https://webpack.js.org/guides/development/#using-webpack-dev-server)
+
+
```

Comment:
- [ ] **[OPTIONAL]:** I suggest you add the Microverse logo to your README file. You can find it [here](https://github.com/microverseinc/readme-template).